book/chapters/git-advanced-branches.qmd

```diff
@@ -1,79 +1,66 @@
 
-The strength of version control is that it permits the researcher to isolate different tracks of work, which can later be merged to create a composite version that contains all changes:
-
-![](../images/github_figures_screenshots/git-collaborative-branches.svg){width=700 fig-align="center"}
-
-
-
-
-- We see branching points and merging points.
-- Main line development is often called main or master.
-- Other than this convention there is nothing special about main or master, it is a branch like any other.
-- Commits form a directed acyclic graph (we have left out the arrows to avoid confusion about the time arrow).
-
-A group of commits that create a single narrative are called a branch. There are different branching strategies, but it is useful to think that a branch tells the story of a feature, e.g. “fast sequence extraction” or “Python interface” or “fixing bug in matrix inversion algorithm”.
+This means that the git history of the project can diverge: different branches will share the first part of their git history but will contain unique commits from the branching point. The strength of git version control is that it facilitates the researcher to isolate different tracks of work, which can later be merged to create a composite version that contains all changes. With merging you add commits from a 'feature' branch to the main branch to work towards a linear git history. More on that later.
 
 To experiment with branches we will create a "fork" from our recipe GitHub repository:
 
 - Go to https://tinyurl.com/repcorecipes
 - Click on the Dropdown menu "Fork" and "Create new fork"
+- Make sure the box "Copy the `main` branch only" is checked
 
-## Creating 
+## Creating a branch
 
 Now we use the web browser to create a new branch in your own recipe repository on Github. You can create a branch in the web browser like this:
 
 1. Go to your repository
-2. Click on "Main" (see screenshot above) and click on "View all branches". You will get an overview of all branches and how actively they are used.
+2. Click on "Main" (see screenshot above) and click on "View all branches". Here you can get an overview of all branches and how actively they are used (now there is just one branch).
 3. Click on "New branch" and give the new branch a name e.g. "cakes".
  
 We have now two branches, both containing the same files in the remote repository. 
 
-Since we only have one branch, the "main" branch, the new branch will automatically created from there. Once you have more than one branch, you can select from which branch you would like to create the new one.
-
-We need to update our local repository on our laptop and "switch" to the new branch to work on a new feature.
+Since we only had one branch, the "main" branch, the new branch will automatically created from the main branch. Once you have more than one branch, you can select from which branch you would like to create the new one.
```

Comment:
```suggestion
Since we only had one branch, the "main" branch, the new branch will automatically be created from the main branch. Once you have more than one branch, you can select from which branch you would like to create the new one.
```